### DISCUSSION TITLE
When running make package install, it says it can’t find foundation/foundation.h

### DISCUSSION THREAD

**Discussion by SakeyJakey** [theos/theos#612] [Questions]
Please help me, I just want to make a tweak, I have tried installing 5 times now.

**Comment by kirb** [comment]
Could you please fill out the [issue template](https://github.com/theos/theos/blob/master/.github/ISSUE_TEMPLATE.md)? It helps us to provide the right assistance based on your configuration.

Review https://git.io/theosinstall and make sure you haven’t missed a step. If you’re using macOS, you need to install Xcode. Otherwise, you need to download an SDK.
